source/openapi3/split.yaml

```diff
@@ -3,1481 +3,7 @@ info:
   title: Zepto API
   contact:
     email: support@zepto.com.au
-  description: >
```

Comment:
This does not appear to be published anywhere, based on my observations of the published readme, my discussions with Sujeet and a general web search. 

But please challenge me on this if you think differently!